docs/install.rst

```diff
@@ -106,15 +113,42 @@ package, depending on how you wish to handle virtual environments:
 
+   With this setup, similar to option 2, you have a single jupyter install that can run kernels from any virtual environment. The main difference is that it can run kernels from *any* virtual environment (not just conda!) but that you have to run an additional line after installing ``ipykernel``  into the environment (``python -m ipykernel install ...``).
+
    .. note:: If you're not using conda to manage your environments, the key idea is to install ``jupyter`` and ``ipywidgets`` in one environment, then install ``ipykernel`` and ``torchvision`` in the same environment as plenoptic, and then run the ``ipykernel install`` command **using the plenoptic environment's python**.
 
+The following table summarizes the advantages and disadvantages of these three choices:
+
+.. list-table::
+   :header-rows: 1
+
+   *  - Method
+      -  Advantages
+      -  Disadvantages
+   *  - 1. Everything in one environment
+      - |:white_check_mark:| Simple
+      - |:x:| Requires lots of hard drive space
+   *  - 2. ``nb_conda_kernels``
+      - |:white_check_mark:| Set up once
+      - |:x:| Initial setup more complicated
+   *  -
+      - |:white_check_mark:| Requires only one jupyter installation
+      -
+   *  -
+      - |:white_check_mark:| Automatically finds new environments with ``ipykernel`` installed
+      -
+   *  - 3. Manual kernel installation
+      - |:white_check_mark:| Flexible: works with any virtual environment setup
+      - |:x:| More complicated
+   *  -
+      - |:white_check_mark:| Requires only one jupyter installation
+      - |:x:| Extra step for each new environment
+
 You can install all of the extra required packages using ``pip install -e .[nb]`` (if you have a local copy of the source code) or ``pip install plenoptic[nb]`` (if you are installing from PyPI). This includes jupyter, and so is equivalent to method 1 above. See the :ref:`optional dependencies section <optional-deps>` for more details.
 
-Once you have jupyter installed and the kernel set up, navigate to the
-``examples/`` directory under this one on your terminal and activate the
-environment you installed jupyter into (``plenoptic`` for 1, ``base`` for 2 or 3),
-then run ``jupyter`` and open up the notebooks. If you followed the second or
-third method, you should be prompted to select your kernel the first time you
-open a notebook: select the one named "plenoptic".
+Running the notebooks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you have jupyter installed and the kernel set up, navigate to plenoptic's ``examples/`` directory on your terminal and activate the environment you installed jupyter into (``plenoptic`` for 1, ``base`` for 2 or 3), then run ``jupyter`` and open up the notebooks. If you followed the second or third method, you should be prompted to select your kernel the first time you open a notebook: select the one named "plenoptic".
```

Comment:
Why not just go to the dir, `conda activate plenoptic`, and get started?

Reply:
you have to activate the environment that jupyter is in, so you'll run `conda activate plenoptic` only if you followed method 1. does that make sense?